sdk/tables/azure-tables/review/tables.api.review.md

```diff
@@ -0,0 +1,695 @@
+}
+
+// @public
+export type SetAccessPolicyOptions = Omit<TableSetAccessPolicyOptionalParams, "tableAcl">;
+
+// @public
+export type SetAccessPolicyResponse = TableSetAccessPolicyHeaders & {
+  _response: coreHttp.HttpResponse & {
+    parsedHeaders: TableSetAccessPolicyHeaders;
+  };
+};
+
+// @public
+export interface SetPropertiesOptions extends coreHttp.OperationOptions {
+  requestId?: string;
+  timeout?: number;
+}
+
+// @public
+export type SetPropertiesResponse = ServiceSetPropertiesHeaders & {
+  _response: coreHttp.HttpResponse & {
+    parsedHeaders: ServiceSetPropertiesHeaders;
+  };
+};
+
+// @public
+export interface SignedIdentifier {
+  accessPolicy: AccessPolicy;
+  id: string;
+}
+
+// @public
+export class TableClient {
+  constructor(url: string, tableName: string, options?: TableServiceClientOptions);
+  createEntity<T>(entity: Entity<T>, options?: CreateEntityOptions): Promise<CreateEntityResponse>;
+  deleteEntity(
+    partitionKey: string,
+    rowKey: string,
+    ifMatch: string,
+    options?: DeleteEntityOptions
+  ): Promise<DeleteEntityResponse>;
+  static fromConnectionString(
+    connectionString: string,
+    tableName: string,
+    options?: TableServiceClientOptions
+  ): TableClient;
+  getAccessPolicy(options?: GetAccessPolicyOptions): Promise<GetAccessPolicyResponse>;
+  getEntity<T extends object>(
+    partitionKey: string,
+    rowKey: string,
+    options?: GetEntityOptions
+  ): Promise<GetEntityResponse<T>>;
+  listEntities<T extends object>(
+    query?: QueryOptions,
+    options?: ListEntitiesOptions
+  ): Promise<ListEntitiesResponse<T>>;
+  setAccessPolicy(
+    acl?: SignedIdentifier[],
+    options?: SetAccessPolicyOptions
+  ): Promise<SetAccessPolicyResponse>;
+  readonly tableName: string;
+  updateEntity<T>(
+    entity: Entity<T>,
+    mode?: UpdateMode,
+    eTag?: string,
+    options?: UpdateEntityOptions
+  ): Promise<UpdateEntityResponse>;
+  upsertEntity<T>(
+    entity: Entity<T>,
+    mode?: UpdateMode,
+    options?: UpsertEntityOptions
+  ): Promise<MergeEntityResponse>;
+}
+
+// @public
+export interface TableCreateHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  preferenceApplied?: string;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableCreateHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  preferenceApplied?: string;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableDeleteEntityHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableDeleteHeaders {
+  clientRequestId?: string;
+  date?: Date;
+  requestId?: string;
+  version?: string;
+}
+
+// @public
+export interface TableEntityContinuationToken {
+  nextPartitionKey?: string;
+  nextRowKey?: string;
+}
+
+// @public
+export interface TableEntityQueryResponse {
+  odataMetadata?: string;
+  value?: {
+    [propertyName: string]: any;
```

Comment:
Note: `value` is being overridden by a generic type above in `ListEntitiesResponse`